helm/cas-provision/templates/deployer/deployerRole.yaml

```
@@ -0,0 +1,181 @@
+# Permissions removed during migration:
+# {APIGroups:[""], Resources:["pods/log"], Verbs:["create" "delete" "deletecollection" "patch" "update"]}
+
+apiVersion: authorization.openshift.io/v1
+kind: Role
+metadata:
+  annotations:
+    openshift.io/description: A user who can deploy applications
+    openshift.io/reconcile-protect: "false"
+  name: {{ .Release.Namespace }}-deployer
+rules:
+  - apiGroups:
+      - image.openshift.io
+    resources:
+      - imagestreamimages
+      - imagestreammappings
+      - imagestreams
+      - imagestreamtags
+    verbs:
+      - get
+      - list
+      - watch
+      - update
+  - apiGroups:
+      - image.openshift.io
+    resources:
+      - imagestreamtags
+    verbs:
+      - delete
+  - apiGroups:
+      - project.openshift.io
+      - ""
+    attributeRestrictions: null
+    resources:
+      - projects
+    verbs:
+      - get
+  - apiGroups:
+      - ""
+    attributeRestrictions: null
+    resources:
+      - replicationcontrollers
+      - persistentvolumeclaims
+      - services
+      - secrets
+      - configmaps
+      - endpoints
+      - pods
+      - pods/exec
+      - pods/log
+    verbs:
+      - watch
+      - list
+      - get
```

You need to separate `pods/log` from the rest, otherwise the role won't be authorized to edit the other resources, e.g. secrets or services, which we definitely need

```suggestion
  - apiGroups:
      - ""
    attributeRestrictions: null
    resources:
      - replicationcontrollers
      - persistentvolumeclaims
      - services
      - secrets
      - configmaps
      - endpoints
      - pods
      - pods/exec
    verbs:
      - watch
      - list
      - get
      - create
      - update
      - patch
      - delete
      - deletecollection
  - apiGroups:
      - ""
    attributeRestrictions: null
    resources:
      - pods/log
    verbs:
      - watch
      - list
      - get
```